Dokan/Dokan/Scenes/CategoryList/CategoryListFooterView/CategoryFooterReusableView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CategoryFooterReusableView.swift
+//  Dokan
+//
+//  Created by Ahmed Nagy on 26/08/2022.
+//
+
+import UIKit
+
+class CategoryFooterReusableView: UICollectionReusableView {
+
+    static let identifier = "CategoryFooterReusableView"
```

Comment:
```suggestion
    static let identifier = "CategoryFooterReusableView"
```
No need for identifier, you can use `IdentifiableView` instead.